oauth2server/proj/settings/default.py

```diff
@@ -81,4 +81,128 @@
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse',
+        },
+        'require_debug_true': {
+            '()': 'django.utils.log.RequireDebugTrue',
+        },
+    },
+    'formatters': {
+        'simple': {
+            'format': '[%(asctime)s] %(levelname)s %(message)s','datefmt': '%Y-%m-%d %H:%M:%S'
+        },
+        'verbose': {
+            'format': '[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s','datefmt': '%Y-%m-%d %H:%M:%S'
+        },
+    },
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            #'filters': ['require_debug_true', 'require_debug_true'],
+            'filters': [],                      # Allow all logs to pass to console
+            'class': 'logging.StreamHandler',
+            'formatter': 'verbose'
+        },
+        'development_logfile': {
+            'level': 'DEBUG',
+            'filters': ['require_debug_true'],
+            'class': 'logging.FileHandler',
+            'filename': 'logs/oauth2_development.log',
+            'formatter': 'verbose'
+        },
+        'production_logfile': {
+            'level': 'WARNING',
+            'filters': ['require_debug_false'],
+            'class': 'logging.FileHandler',
+            'filename': 'logs/oauth2_production.log',
+            'formatter': 'simple'
+        },
+        'remote_logfile': {
+            'level': 'DEBUG',
+            'filters': ['require_debug_false', 'require_debug_true'],
+            'class': 'logging.FileHandler',
+            'filename': 'logs/oauth2_remote.log',
+            'formatter': 'simple'
+        },
+        'proj_logfile': {
+            'level': 'DEBUG',
+            #'filters': ['require_debug_false','require_debug_true'],
+            'filters':[],                           # Allow all 'proj' related logs to be sent to this project folder
+            'class': 'logging.FileHandler',
+            'filename': 'logs/oauth2_proj.log',
+            'formatter': 'simple'
+        },
+    },
+    'loggers': {
+        # This is our main log handler. It catches all logs within apps.*.*
+        'apps': {
+            'handlers': ['console', 'production_logfile', 'development_logfile'],
+            'level': 'DEBUG',
+        },
+        # This defines a handler for the namespace proj.*.*
+        'proj': {
+            'handlers': ['console', 'development_logfile', 'proj_logfile'],
+        },
+        # Our remote handler is used for logging anything we want to be logged while the app is running remotely
+        'remote': {
+            'handlers': ['console', 'remote_logfile'],
+        },
+        # This is our default django handler loggs
+        'django': {
+            'handlers': ['console', 'development_logfile', 'production_logfile'],
+        },
+        # This is our default warnings log
+        'py.warnings': {
+            'handlers': ['console', 'development_logfile'],
+        },
+    }
+}
+
+
+
+
+
```

Comment:
Should the commented blocks be removed? Not sure what the policy is with commented out code blocks.